docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+* Always enforce default tier preference {es-pull}79751[#79751] (issue: {es-issue}76147[#76147])
+* Validate that snapshot repository exists for ILM policies at creation/update time {es-pull}78468[#78468] (issues: {es-issue}72957[#72957], {es-issue}77657[#77657])
+* Default `ENFORCE_DEFAULT_TIER_PREFERENCE` to `true` {es-pull}79275[#79275] (issues: {es-issue}76147[#76147], {es-issue}79210[#79210])
+
+Features/Indices APIs::
+* Change prefer_v2_templates parameter to default to true {es-pull}55489[#55489] (issues: {es-issue}53101[#53101], {es-issue}55411[#55411])
+* Remove deprecated `_upgrade` API {es-pull}64732[#64732] (issue: {es-issue}21337[#21337])
+* Remove local parameter for get field mapping request {es-pull}55100[#55100] (issue: {es-issue}55099[#55099])
+* Remove `include_type_name` parameter from REST layer {es-pull}48632[#48632] (issue: {es-issue}41059[#41059])
+* Remove the `template` field in index templates {es-pull}49460[#49460] (issue: {es-issue}21009[#21009])
+* Remove endpoint for freezing indices {es-pull}78918[#78918] (issues: {es-issue}70192[#70192], {es-issue}77273[#77273])
+
+Features/Watcher::
+* Move watcher history to data stream {es-pull}64252[#64252]
+
+Geo::
+* Disallow creating `geo_shape` mappings with deprecated parameters {es-pull}70850[#70850] (issue: {es-issue}32039[#32039])
+* Remove bounding box query `type` parameter {es-pull}74536[#74536]
+
+Infra/Circuit Breakers::
+* Fixed synchronizing inflight breaker with internal variable {es-pull}40878[#40878]
+
+Infra/Core::
+* Fail when using multiple data paths {es-pull}72184[#72184] (issue: {es-issue}71205[#71205])
+* Limit processors by available processors {es-pull}44894[#44894] (issue: {es-issue}44889[#44889])
+* Remove `nodes/0` folder prefix from data path {es-pull}42489[#42489]
+* Remove `bootstrap.system_call_filter` setting {es-pull}72848[#72848]
+* Remove `fixed_auto_queue_size` threadpool type {es-pull}52280[#52280]
+* Remove `node.max_local_storage_nodes` {es-pull}42428[#42428] (issue: {es-issue}42426[#42426])
+* Remove camel case named formats {es-pull}60044[#60044]
+* Remove legacy role settings {es-pull}71163[#71163] (issues: {es-issue}54998[#54998], {es-issue}66409[#66409], {es-issue}71143[#71143])
+* Remove `processors` setting {es-pull}45905[#45905] (issue: {es-issue}45855[#45855])
+* Remove the `local` parameter of `/_cat/nodes` {es-pull}50594[#50594] (issues: {es-issue}50088[#50088], {es-issue}50499[#50499])
+* Remove the listener thread pool {es-pull}53314[#53314] (issue: {es-issue}53049[#53049])
+* Remove the node local storage setting {es-pull}54381[#54381] (issue: {es-issue}54374[#54374])
+* Remove the `pidfile` setting {es-pull}45940[#45940] (issue: {es-issue}45938[#45938])
+* Removes `week_year` date format {es-pull}63384[#63384] (issue: {es-issue}60707[#60707])
+* Fail index creation using custom data path {es-pull}76792[#76792] (issue: {es-issue}73168[#73168])
```

Comment:
This PR has also been reverted.

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c